File: CodeFreak1/ClientApp/src/app/Security/Dtos/users-view-model.ts
import { RequestStatus } from "./request-status";

export class UsersViewModel extends RequestStatus {
  public UserId: string;
  public Login: string;
  public Password: string;
  public Name: string;
  public Email: string;
  public CreatedOn: Date;
  public ModifiedOn: Date;
  public IsActive: boolean;
  public CreatedBy: string;
  public ModifieBy: string;
  public DateOfBirth: Date;
}

File: CodeFreak1/ViewModel/UsersViewModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace CodeFreak1.ViewModel
{
    public class UsersViewModel : RequestStatus
    {
        public Guid UserId { get; set; }
        public string Login { get; set; }
        public string Password { get; set; }
        public string Name { get; set; }
        public string Email { get; set; }
        public DateTime CreatedOn { get; set; }
        public DateTime? ModifiedOn { get; set; }
        public bool? IsActive { get; set; }
        public Guid? CreatedBy { get; set; }
        public Guid? ModifieBy { get; set; }
    }
}
